guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -4,31 +4,30 @@
 Use this procedure to create a user.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-user_{context}[].
 
-If a new user is not assigned to an organization, their access is limited.
-To grant systems rights to users, assign them to an organization.
-The next time the user logs on to {Project}, the user's account has the correct system rights.
-
 .Procedure
 . In the {ProjectWebUI}, navigate to *Administer* > *Users*.
 . Click *Create User*.
-. In the *Login* field, enter a username for the user.
-. In the *Firstname* and *Lastname* fields, enter the real first name and last name of the user.
-. In the *Mail* field, enter the user’s email address.
-. In the *Description* field, add a description of the new user.
-. Select a specific language for the user from the *Language* list.
-. Select a timezone for the user from the *Timezone* list.
-+
-By default, {ProjectServer} uses the language and timezone settings of the user’s browser.
-. Set a password for the user:
-.. From the *Authorized by* list, select the source by which the user is authenticated.
-** *INTERNAL*: to enable the user to be managed inside {ProjectServer}.
-** *EXTERNAL*: to configure external authentication as described in {InstallingServerDocURL}Configuring_External_Authentication_{project-context}[Configuring External Authentication] in _{InstallingServerDocTitle}_.
-.. Enter an initial password for the user in the *Password* field and the *Verify* field.
+. Enter the account details for the new user.
 . Click *Submit* to create the user.
 
+For example, you can specify the following user account details:
+
+* On the *User* tab, you must set a password for the user.
+Select the source that the user is authenticated by from the *Authorized by* list:
+** Select *INTERNAL* to manage the user inside {ProjectServer}.
+** Select *EXTERNAL* to manage the user with external authentication as described in {InstallingServerDocURL}Configuring_External_Authentication_{project-context}[Configuring External Authentication] in _{InstallingServerDocTitle}_.
+* On the *Organizations* tab, select an organization for the user.
+You can use the *Default on login* drop-down list to select which organization is selected for the user after they log in.
```

Comment:
Our style guides say to use "list" instead of "drop-down list".

"to select which organization is selected for the user after they log in". That's a lot of words. Maybe this: 

`Optional: Select a default organization from the *Default on login* list.`

Reply:
Shortening it your way would alter the meaning in a way that I don't quite like. But I think I've found another way to reword this -- it's still kind of long, but there is not word repetition or passive voice.

About using `Optional:` here: Whether or not a web UI field is optional is indicated by `*`. I'd prefer not to duplicate that in the guide.